src/inmanta/app.py

```diff
@@ -373,6 +373,7 @@ def export_parser_config(parser):
         action="store_true",
         default=False,
     )
+    parser.add_argument("--export-plugin", dest="export_plugin", help="Use this export plugin.", default=None)
```

Comment:
I would expand the help message and also update the help message of the configuration.

```suggestion
    parser.add_argument("--export-plugin", dest="export_plugin", help="Only use this export plugin. This option also disables the execution of the plugins listed in the configuration file in the export setting.", default=None)
```

A similar update is needed in the config definition of config.export